Invalid main and types path in package.json (Error: Cannot find module)

The `main` and `types` paths `package.json` are incorrect (at-least in the version published to NPM).

They are set to:

```json
"main": "./dist/index.js",
"types": "./dist/index.d.ts",
```

But those files are are actually at:

```
./dist/src/main/index.js
./dist/src/main/index.d.ts
```

When attempting to load the module I get this error.

> Error: Cannot find module '.../node_modules/resedit/dist/index.js'. Please verify that the package.json has a valid "main" entry

I can still use it if I use a deep-import, but it would be nice if this were fixed properly.

```js
require('resedit/dist/src/main')
```

PS. This is a pretty cool module. With a little scripting I think this can be used as a platform-independent replacement for rcedit.

--- a/tsconfig.app.json
+++ b/tsconfig.app.json
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": "./src/main",
+    "baseUrl": "./",
     "declaration": true,
     "lib": [ "es5", "es2015.collection" ],
     "module": "commonjs",
@@ -15,7 +15,7 @@
         "../../node_modules/*"
       ]
     },
-    "rootDir": "./",
+    "rootDir": "./src/main",
     "skipLibCheck": true,
     "strict": true,
     "target": "es5",

--- a/tsconfig.test.json
+++ b/tsconfig.test.json
@@ -1,6 +1,8 @@
 {
   "extends": "./tsconfig.app.json",
   "compilerOptions": {
+    "baseUrl": "./src/main",
+    "rootDir": "./",
     "paths": {
       "@/*": [
         "../../src/main/*"

